src/e2e/config/webdriverio.config.js

```diff
@@ -9,25 +9,27 @@ const user = process.env.BROWSERSTACK_USERNAME
 const config = {
   user,
   key: process.env.BROWSERSTACK_ACCESS_KEY,
-
   capabilities: {
     platformName: 'iOS',
-    platformVersion: '14.4',
-    deviceName: 'iPhone 12',
     browserName: 'Safari',
+    device: 'iPhone 11',
+    osVersion: '13.3',
```

Comment:
Can you add a comment here that describes why the device and osVersion is important for the tests? Thanks!